Formula/quantlib.rb

```diff
@@ -38,12 +37,20 @@ class Quantlib < Formula
   def install
     ENV.cxx11 if build.cxx11?
     if build.head?
-      Dir.chdir "QuantLib"
       system "./autogen.sh"
     end
+    if MacOS.version >= :mavericks
+      ENV.append "LDFLAGS", "-stdlib=libstdc++ -mmacosx-version-min=10.6"
+      ENV.append "CXXFLAGS", "-stdlib=libstdc++ -mmacosx-version-min=10.6"
```

Comment:
I think `stdlib=libstdc++` is because libc++ became default by mavericks. I am not so sure why ` -mmacosx-version-min=10.6` is needed. I didn't test if it would work w/o it.

Quantlib advertises this solution. Currently, this formula is virtually useless.